wp-content/themes/pub/wporg-learn-2020/functions.php

```diff
@@ -58,16 +58,20 @@ function wporg_learn_scripts() {
 		true
```

Comment:
This parameter being `true` tells WordPress to load the script in the footer. Without `wp_footer` being called this never happens. The `wporg-filters` script below is enqueued the same way and never loads.

https://developer.wordpress.org/reference/functions/wp_enqueue_script/